account_anglo_saxon_stock_move/model/invoice.py

```diff
@@ -235,10 +232,10 @@ def reconcile_stock_accrual(self, rec_ids, query_col, do_commit=False):
         query = self._get_accrual_query(query_col, 'query4', query_params)
         self._cr.execute(query)
 
-        if query_col == 'sale_id':
-            obj = self.env['sale.order']
-        elif query_col == 'purchase_id':
-            obj = self.env['purchase.order']
+        obj = self.env[
```

Comment:
single row

Reply:
What do you mean?

Reply:
all the statement in in single line instead of

Reply:
Done. Also moved to a separate method, because this same code was duplicated several times.